src/main/java/org/assertj/core/api/AbstractLocalDateTimeAssert.java

```diff
@@ -229,6 +222,15 @@ public SELF isAfter(String localDateTimeAsString) {
     return isAfter(parse(localDateTimeAsString));
   }
 
+  public SELF isEqualTo(LocalDateTime expected) {
```

Comment:
add javadoc and tests in `LocalDateTimeAssert_isEqualTo_Test` showing the new behavior (based on ChronoLocalDateTimeComparator).

Reply:
same comment for `isNotEqualTo`

Reply:
these comments also applies to `AbstractOffsetDateTimeAssert` and `AbstractZonedDateTimeAssert`.

Reply:
Could you please clarify what kind of tests you are missing exactly?

In one of your later comments you state that it is enough to test that the delegation happens to `comparables` (and `objects`, I assume). There are already tests which verify that the default comparator is `ChronoLocalDateTimeComparator` (`LocalDateTimeAssert_defaultComparator_Test`), and though the comparator is used in this assertion, it is not possible to use objects with type other than `LocalDateTime` as parameter, so the comparator is not utilized to its full extent.

What is your opinion about accepting `Object` as parameter (so API does not break) and delegating to `comparables` only if the expected value is instance of `ChronoLocalDateTime` (or `LocalDateTime`) and to `objects` otherwise?
Ofc, using `ChronoLocalDateTime` would break symmetry since there is no `ChronoLocalDateTimeAssert` currently, i.e.
```
LocalDateTime now = LocalDateTime.now();
ChronoLocalDateTime<JapaneseDate> inJapanese = JapaneseChronology.INSTANCE.localDateTime(now);
assertThat(now).isEqualTo(inJapanese); // succeed
assertThat(inJapanese).isEqualTo(now); // fails
```

Reply:
:eyes: